lms/templates/instructor/instructor_dashboard_2/membership.html

```diff
@@ -241,9 +250,138 @@
       data-add-button-label="${_("Add Community TA")}"
     ></div>
   %endif
-
 </div>
 
+
+<div
+    class="email-lists-management"
+    data-query-endpoint="${ section_data['get_single_query']}"
+    data-total-endpoint="${ section_data['get_all_students']}"
+    data-temp-queries-endpoint="${ section_data['get_temp_queries']}"
+    data-delete-bulk-temp-endpoint="${ section_data['delete_bulk_temp_query']}"
+    data-delete-saved-endpoint="${ section_data['delete_saved_query']}"
+    data-delete-temp-endpoint="${ section_data['delete_temp_query']}"
+>
+    <div class="email-widget">
+    <div class="header">
+        <h2 class="title">Email Distribution</h2>
+    </div>
+    <label for="beginning_specific">
+        Select criteria on which to generate a list of student emails.
+        Please clear the active queries or load your saved work if you
+        are inactive for more than 30 minutes, otherwise queries might
+        be inaccurate. If the status gets stuck at "working" for a long
+        time, you may come back to the page later.
+    </label>
+    <br/>
+    <div class="email-list">
+        <div class="email-list-container"
+             data-label ="Inclusion"
+             data-selections="AND <> NOT <> OR">
+        </div>
+        <div class="email-list-container"
+             data-label ="Select a Type"
+             data-selections="Section<>Problem">
+        </div>
+        <div class="problem_specific">
+            <div class="email-list-container"
+                 data-rolename="staff"
+                 data-label ="Select Problem"
+                 data-selections="Loading problems..."
+                 data-list-endpoint="${ section_data['list_course_problems'] }">
+            </div>
+        </div>
+        <div class="problem_specific">
+            <div class="email-list-container"
+             data-label ="Filter On"
+             data-selections="Opened <>
+                              Completed <>
+                              Not Opened <>
+                              Not Completed">
+            </div>
+        </div>
+        <div class="section_specific">
+             <div class="email-list-container"
+                 data-rolename="staff"
+                 data-label ="Select Section"
+                 data-selections="Loading sections..."
+                 data-list-endpoint="${ section_data['list_course_sections'] }">
+            </div>
+        </div>
+        <div class="section_specific">
+          <div class="email-list-container"
+               data-label ="Filter On"
```

Comment:
All of these `data-label` attributes have a superfluous whitespace before the assignment operator.